app/_how-tos/use-ai-prompt-template-plugin.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Use the AI Prompt Template plugin
+content_type: how_to
+related_resources:
+  - text: AI Gateway
+    url: /ai-gateway/
+  - text: AI Proxy
+    url: /plugins/ai-proxy/
+  - text: AI Prompt Template
+    url: /plugins/ai-prompt-template/
+
+description: |
+  Configure the AI Proxy plugin to route requests to a model provider like Mistral, then define reusable templates with the AI Prompt Template plugin to enforce consistent prompt formatting for tasks like summarization, code explanation, and Q&A.
+
+tldr:
+  q: How do I use prompt templates with AI Gateway?
+  a: Configure the [AI Proxy](/plugins/ai-proxy/) plugin to route traffic, then use the [AI Prompt Template](/plugins/ai-prompt-template/) plugin to define and enforce reusable prompt formats.
+
+products:
+  - gateway
+  - ai-gateway
+
+works_on:
+  - on-prem
+  - konnect
+
+min_version:
+  gateway: '3.6'
+
+plugins:
+  - ai-proxy
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+  - ai
+  - openai
+
+tools:
+  - deck
+
+prereqs:
+  inline:
+    - title: Mistral
+      include_content: prereqs/mistral
+      icon_url: /assets/icons/mistral.svg
+  entities:
+    services:
+      - example-service
+    routes:
+      - example-route
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
+
+## Configure the AI Proxy plugin
+
+Start by configuring the AI Proxy plugin to route prompts to Mistral AI.
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      config:
+        route_type: llm/v1/chat
+        auth:
+          header_name: Authorization
+          header_value: Bearer ${key}
+        model:
+          provider: mistral
+          name: mistral-tiny
+          options:
+            mistral_format: openai
+            upstream_url: https://api.mistral.ai/v1/chat/completions
+
+variables:
+  key:
+    value: $MISTRAL_API_KEY
+    description: The API key to use to connect to Mistral.
+{% endentity_examples %}
+
+
+## Configure the AI Prompt Template plugin
+
+Now, we can configure the AI Prompt Template plugin with predefined, reusable prompt templates for common tasks. The plugin will automatically [block all untemplated requests](/how-to/use-ai-prompt-template-plugin/#denied-prompts) via `allow_untemplated_requests: false` setting.
```

Comment:
```suggestion
Now, we can configure the AI Prompt Template plugin with predefined, reusable prompt templates for common tasks. This allows users to fill in the blanks with variable placeholders (`{{variable}}`).

The plugin will automatically [block all untemplated requests](/how-to/use-ai-prompt-template-plugin/#denied-prompts) via `allow_untemplated_requests: false` setting.
```
I liked the description from the plugin, it helped me understand what this was doing.